### PR TITLE
Align bare-metal-install-* docs with clr-installer verify-iso-integrity

### LINKED ISSUE #1185: Align bare-metal-install-* with clr-installer new feature, verify-iso-integrity
**Describe the error/improvement to an existing document or image**

Add new section `Verify integrity of installer media` in: 
https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-desktop.html
https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html

Align CL Docs with clr-installer with release of:
- PR: https://github.com/clearlinux/clr-installer/pull/737 
- Issue: https://github.com/clearlinux/clr-installer/issues/490

**Screenshots**
TBD: New screenshot forthcoming to replace Figure 1.

**Environment (please complete the following):**
 - Clear Linux OS version: 33210

**Additional context**

### PR DESCRIPTION
**Pull request by mvincerx**
- New clr-installer feature, uses checksum to verify ISO image
- See also: https://github.com/clearlinux/clr-installer/pull/737
- Closes #1185

-TBD: Waiting on screenshot from @karthikprabhuvinod to replace Figure 1.

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

**Comment by mvincerx** [comment]
> I think the 'Verify integrity of installer media (optional)' should come before the 'Install from Live Image'. Right now it seems to be injected in the middle of the live image content.

In theory, I understand that. In practice, however, if you walk through the steps, you won't see the option to `Verify ISO Integrity` until after you reboot the target system.  That's why I make a reference to the new section on line 65 (`bare-metal-install-desktop`). 

IOW: How can you follow the option to `Verify ISO Integrity` before you reboot the USB with flashed ISO image?

**Comment by mdhorn** [comment]
> > I think the 'Verify integrity of installer media (optional)' should come before the 'Install from Live Image'. Right now it seems to be injected in the middle of the live image content.
> 
> In theory, I understand that. In practice, however, if you walk through the steps, you won't see the option to `Verify ISO Integrity` until after you reboot the target system. That's why I make a reference to the new section on line 65 (`bare-metal-install-desktop`).
> 
> IOW: How can you follow the option to `Verify ISO Integrity` before you reboot the USB with flashed ISO image?

Perhaps GitHub doesn't render what the final doc will look like correctly, but it should the new section at the same level as "Install from Live Image" which is not right. Can you tell me how to view that the final results will be?

File: source/get-started/bare-metal-install-server.rst [hunk 1]
```diff
@@ -70,8 +70,28 @@ Follow these steps to install |CL| on the target system:
...
+:guilabel:`clr-installer` implants checksums, which are verified during 
+early boot stage as part of :command:`initrd`. 
+ 
+#. Select :guilabel:`Verify ISO Integrity`. The media will be validated. 
+
+#. If the check passes, it will boot into the live image. 
+
+#. If the check fails, a failure message appears. 
+   
+   #. Follow `Preliminary Steps`_.
```
Comment (karthikprabhuvinod) [inline source/get-started/bare-metal-install-server.rst:93]: Isnt this supposed to point to a section which says something like trying reburning your image to the media?? Are you yet to write that?

**Comment by mvincerx** [comment]
@mdhorn, @karthikprabhuvinod, see my latest email for clarification.

File: source/get-started/bare-metal-install-server.rst (hunk 1 above)
Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: `Preliminary steps` is a hard link. Basically, if the image fails the checksum, the user has to return to the Preliminary steps and download a new image.  In my latest commit, I reworded that sentence as:  

```
Restart the process at `Preliminary Steps`_. 
```

File: source/get-started/bare-metal-install-desktop.rst [hunk 2]
```diff
@@ -61,28 +62,40 @@ these steps.
...
 
-Software (optional)
-===================
+Verify integrity of installer media (optional)
+==============================================
 
-Explore |CL| bundles and other software. Double-click the
-:guilabel:`Software` icon from the Activities menu, shown in Figure 2.
-Ensure a network connection exists before launching the Software application.
+Use :guilabel:`Verify ISO Integrity` to verify the checksum of
```
Comment (karthikprabhuvinod) [inline source/get-started/bare-metal-install-desktop.rst:85]: Can we reword this a little. 

```suggestion
Use :guilabel:`Verify ISO Integrity` to verify the checksum of 
the image burned to the installer media. The checksums helps ensure the ISO is un-corrupted (see Figure 1).  For every ISO generated, the :guilabel:`clr-installer` implants checksums, which are verified during 
early boot stage as part of :command:`initrd`.
```

File: source/get-started/bare-metal-install-server.rst [hunk 3]
```diff
@@ -62,15 +62,38 @@ Follow these steps to install |CL| on the target system:
...
       :scale: 100%
       :alt: Clear Linux OS Installer boot menu
 
       Figure 1: Clear Linux OS Installer boot menu
 
-#. With :guilabel:`Clear Linux OS` highlighted, select :kbd:`Enter`.
+Verify integrity of installer media (optional)
+==============================================
+
+Use :guilabel:`Verify ISO Integrity` to verify the checksum of
```
Comment (karthikprabhuvinod) [inline source/get-started/bare-metal-install-server.rst:83]: same as above

**Comment by karthikprabhuvinod** [comment]
@mvincerx : Thank for your email and those snapshots. I also learnt to build the doc myself because you have perfect instructions for those.

I checked the structure it looks good to me except the last comment i made. 

I wanted to also let you know another detail if that was missed yesterday in the discussion: If on those boot menu screens if user doesn't take any action, it selects the clear installation by default. I am not sure if you want to highlight that.

**Comment by mvincerx** [comment]
@karthikprabhuvinod See latest commit.

**Comment by mvincerx** [comment]
@mdhorn, please review/approve. Thanks

File: source/get-started/bare-metal-install-desktop.rst [hunk 4]
```diff
@@ -61,28 +62,44 @@ these steps.
...
+      .. figure:: /_figures/bare-metal-install-desktop/bare-metal-install-desktop-01.png
+         :scale: 100%
+         :alt: Clear Linux OS in boot menu
 
-      Figure 1: Clear Linux OS in boot menu
+         Figure 1: Clear Linux OS in boot menu
+
+      .. note::
+
+         If no action is taken, installation starts by default.
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-desktop.rst:82]: Technically, the installation will not start, it just boots into the live image.

```suggestion
         If no action is taken, the live image starts by default.
```

File: source/get-started/bare-metal-install-server.rst [hunk 5]
```diff
@@ -62,15 +62,42 @@ Follow these steps to install |CL| on the target system:
...
    .. figure:: /_figures/bare-metal-install-server/bare-metal-install-server-01.png
       :scale: 100%
       :alt: Clear Linux OS Installer boot menu
 
       Figure 1: Clear Linux OS Installer boot menu
 
-#. With :guilabel:`Clear Linux OS` highlighted, select :kbd:`Enter`.
+   .. note::
+
+      If no action is taken, installation starts by default.
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-server.rst:82]: Technically, the installation will not start, it just boots into the live image.

```suggestion
         If no action is taken, the live image starts by default.
```

File: source/get-started/bare-metal-install-desktop.rst (hunk 4 above)
Comment (mvincerx) [reply source/get-started/bare-metal-install-desktop.rst]: @mdhorn Thanks for the attention to wording. I applied your suggestions. See latest commit.

File: source/get-started/bare-metal-install-server.rst (hunk 5 above)
Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: @mdhorn Thanks for the attention to wording. I applied your suggestions. See latest commit.

**Pull request merged**